src/Avalonia.Native/AvaloniaNativeMenuExporter.cs

```diff
@@ -98,7 +98,8 @@ private void DoLayoutReset(bool forceUpdate = false)
                         SetMenu(_nativeWindow, _menu);
                     }
                 }
-
+                OnIsNativeMenuExportedChanged?
```

Comment:
CS0067 should be completely ignored since compiler triggers it when an interface forces you to implement it in a class that has no use for it. Any changes made solely to satisfy that compiler warning should be reverted

Reply:
I reverted it.